4.7.1/build/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# escape=`
+
+FROM microsoft/dotnet-framework:4.7.1-windowsservercore
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+# Install Git
+RUN Invoke-WebRequest -UseBasicParsing https://github.com/git-for-windows/git/releases/download/v2.14.3.windows.1/MinGit-2.14.3-64-bit.zip -outfile git.zip; `
```

Comment:
It's probably been discussed before, but why add git?

Reply:
`git` was included because it is a common ask.  I am open to feedback on whether or not it should be included.

Reply:
I will note this is inconsistent with the .NET Core Windows Dockerfiles - perhaps we should remove it to begin with.

Reply:
Scenario-wise, I think it makes sense to include git. I've used that in our .NET Core SDK images. It enables a certain style of deployment. Certainly, it only does that for one SCM, but I think that's OK. People are free to add their own, but git is what the team uses and many customers use, so it feels OK to include just it.

I looked at [buildpack-deps](https://hub.docker.com/_/buildpack-deps/). Nothing else popped out to me to add that makes sense on Windows. We could consider a component like [curl](https://curl.haxx.se/download.html). As you can see, Powershell provides analogous functionality. If we wanted to avoid setting a specific shell, then curl would be more attractive.

Reply:
I think we should start off without including git.  I think the usage scenarios for it are limited.  I would rather be in the situation where we get requests to add it.  If we release with it, it will be difficult to remove it down the road.  Lastly, we don’t have it in the .NET Core SDK images and I don’t recall anyone asking for it.  I talked to @richlander about this and he is fine with excluding it to begin with.